### ISSUE TITLE
Export `class` variable

### ISSUE THREAD

**Issue by kwaa** [janosh/svelte-bricks#8]
I'm using:
```svelte
<div class="mx-4 sm:mx-8 md:my-4 lg:mx-16 lg:my-8 xl:mx-32 xl:my-16">
  <Masonry
    {items}
    ...
  >
    <Some {item} />
  </Masonry>
</div>
```

If export a `class` variable, that can be reduced to:

```svelte
<Masonry
  class="mx-4 sm:mx-8 md:my-4 lg:mx-16 lg:my-8 xl:mx-32 xl:my-16"
  {items}
  ...
>
  <Some {item} />
</Masonry>
```

This can be achieved by doing:

```svelte
<script>
  let className = ``
  export { className as class }
  ...
</script>
<div
  class="masonry {className}"
  ...
>
...
</div>
```

**Comment by janosh** [comment]
@kwaa Thanks for the suggestion. Sounds like a good idea. 👍

**Comment by janosh** [comment]
Published as [v0.1.6](https://www.npmjs.com/package/svelte-bricks/v/0.1.6).
